docs/src/@primer/gatsby-theme-doctocat/components/head.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+import useSiteMetadata from '@primer/gatsby-theme-doctocat/src/use-site-metadata'
+
+function Head(props) {
+  const siteMetadata = useSiteMetadata()
+  const title = props.title
+    ? `${props.title} | ${siteMetadata.title}`
+    : siteMetadata.title
+  const description = props.description || siteMetadata.description
+
+  return (
+    <Helmet>
+      <title>{title}</title>
+      <meta name="description" content={description} />
+      <meta property="og:title" content={title} />
+      <meta property="og:description" content={description} />
+      <meta property="og:image" content={siteMetadata.imageUrl} />
+      <meta property="twitter:card" content="summary_large_image" />
+      <link href="https://unpkg.com/@primer/css-next@canary/dist/primer.css" rel="stylesheet" />
```

Comment:
Looks like including this CSS file changes the font size of all the text on the page from `16px` to `14px`. It be great if we could prevent that somehow.